samwise.js

```diff
@@ -3,17 +3,16 @@
  *
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.